.circleci/config.yml

```
@@ -44,7 +44,7 @@ defaults: &defaults
     AWS_REGION_ARTIFACTS: eu-central-1
   working_directory: /tmp/material-ui
   docker:
-    - image: cimg/node:14.20
+    - image: cimg/node:16.17
```

```suggestion
    - image: cimg/node:14.20
```
I rather not change this because of https://github.com/mui/material-ui/pull/34178#issuecomment-1238073177

---

This was done only to test if the new Argos SDK works. I don't intend to update Node in this PR.